boards/samd21-xpro/include/periph_conf.h

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (C) 2017 Travis Griggs <travisgriggs@gmail.com>
+ * Copyright (C) 2017 Dan Evans <photonthunder@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_samd21-xpro
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for the Atmel SAM D21 Xplained
+ *              Pro board
+ *
+ * @author      Travis Griggs <travisgriggs@gmail.com>
+ * @author      Dan Evans <photonthunder@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#include "cpu.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   External oscillator and clock configuration
+ *
+ * For selection of the used CORECLOCK, we have implemented two choices:
```

Comment:
Reworded to remove the "we" as suggested. Also reworded another "we" verbiage further down.